doc/release-notes.rst

```diff
@@ -5,11 +5,276 @@
 v12.1.0 Luminous (RC)
 ====================
 
-This is the  first release candidate for Luminous, the next long term stable release.
+This is the first release candidate for Luminous, the next long term
+stable release.
+
+Ceph Luminous (v12.2.0) will be the foundation for the next long-term
+stable release series.  There have been major changes since Kraken
+(v11.2.z) and Jewel (v10.2.z), and the upgrade process is non-trivial.
+Please read these release notes carefully.
 
 Major Changes from Kraken
 -------------------------
 
+- *General*:
+
+  * Ceph now has a simple, built-in web-based dashboard for monitoring
+    cluster status.  FIXME DOCS.
+  
+- *RADOS*:
+
+  * *BlueStore*:
+    - The new *BlueStore* backend for *ceph-osd* is now stable and the new
+      default for newly created OSDs.  BlueStore manages data stored by each OSD
+      by directly managing the physical HDDs or SSDs without the use of an
+      intervening file system like XFS.  This provides greater performance
+      and features. FIXME DOCS
+    - BlueStore supports *full data and metadata checksums* of all
+      data stored by Ceph.
+    - BlueStore supports inline compression using zlib, snappy, or LZ4.  (Ceph
+      also supports zstd for RGW compression but zstd is not recommended for
+      BlueStore for performance reasons.)  FIXME DOCS
+
+  * *Erasure coded* pools now have full support for *overwrites*,
+    allowing them to be used with RBD and CephFS.
+
```

Comment:
all the osd sleep settings now work. osd_recovery_sleep is probably the most important to call out